src/test/java/training/AppTest.java

```diff
@@ -1,14 +1,77 @@
-/*
- * This Java source file was generated by the Gradle 'init' task.
- */
-package training;
-
-import org.junit.Test;
-import static org.junit.Assert.*;
-
-public class AppTest {
-    @Test public void testAppHasAGreeting() {
-        App classUnderTest = new App();
-        assertNotNull("app should have a greeting", classUnderTest.getGreeting());
+package Training;
+
+import java.util.Scanner;
+
+abstract class AppTest {
+  abstract void findTriangle(double b, double h);
+  abstract void findRectangle(double l, double b);
+  abstract void findSquare(double s);
+  abstract void findCircle(double r);
+}
+
+class findArea extends AppTest {
+
+  void findTriangle(double b, double h)
+  {
+    double area = (b*h)/2;
+    System.out.println("Area of Triangle: "+area);
+  }
+
+  void findRectangle(double l, double b)
+  {
+    double area = l*b;
+    System.out.println("Area of Rectangle: "+area);
+  }
+
+  void findSquare(double s)
+  {
+    double area = s*s;
+    System.out.println("Area of Square: "+area);
+  }
+
+  void findCircle(double r)
+  {
+    double area = 3.14*r*r;
+    System.out.println("Area of Circle: "+area);
+  }
+}
+
+class area {
+    public static void main(String args[]) {
+        double l, b, h, r, s;
+        findArea area = new findArea();
+        Scanner get = new Scanner(System.in);
+        System.out.println("Enter a shape:1.triangle,2.rectangle,3.square,4.circle:");
+
+        Scanner kb = new Scanner(System.in);
+        int choice = kb.nextInt();
+        switch (choice) {
```

Comment:
If you are going to create proper classes and use inheritance property then you will simplify this code currently the way you have used it has become more complicated.

Reply:
i used switch so that user can find the area of desired shape.but yeah i will try to make it more simple and update it as soon as possible